examples/raytracing/example_raytracing.cpp

```diff
@@ -21,56 +21,56 @@
 #include <vector>
 
 template <typename MemorySpace>
-struct Primitives
+struct SpheresToBoxes
 {
   Kokkos::View<ArborX::Sphere *, MemorySpace> _spheres;
 };
 
 template <typename MemorySpace>
-struct ArborX::AccessTraits<Primitives<MemorySpace>, ArborX::PrimitivesTag>
+struct ArborX::AccessTraits<SpheresToBoxes<MemorySpace>, ArborX::PrimitivesTag>
 {
   using memory_space = MemorySpace;
 
   KOKKOS_FUNCTION static std::size_t
-  size(const Primitives<MemorySpace> &primitives)
+  size(const SpheresToBoxes<MemorySpace> &stob)
   {
-    return primitives._spheres.extent(0);
+    return stob._spheres.extent(0);
   }
   KOKKOS_FUNCTION static ArborX::Box
-  get(Primitives<MemorySpace> const &primitives, std::size_t const i)
+  get(SpheresToBoxes<MemorySpace> const &stobs, std::size_t const i)
   {
-    auto const &sphere = primitives._spheres(i);
-    auto const &c = sphere.centroid();
-    auto const &r = sphere.radius();
+    auto const &sphere = stobs._spheres(i);
+    auto const c = sphere.centroid();
```

Comment:
No, this was wrongly changed. Change back to `auto const &c = sphere.centroid();`, as it is a complex `Point` struct.